mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -1066,7 +1065,6 @@ def compute_next_token(current_size):
     def log_batch(self, run_id, metrics, params, tags):
         _validate_run_id(run_id)
         _validate_batch_log_data(metrics, params, tags)
-        _validate_batch_log_limits(metrics, params, tags)
```

Comment:
I think we should still keep this validation in the backend. Can we revert this change?